ERCS/erc-7776.md

```diff
@@ -0,0 +1,554 @@
+---
+eip: 7776
+title: Transparent Financial Statements
+description: A.P.I. for Protocols to Ensure Open Transparent Financial Data for all Investors.
```

Comment:
First, we should cut out anything that is common to most ERCs:

```suggestion
description: Open Transparent Financial Data for all Investors.
```

Next, we need to add more details that aren't already in your title. You get about 140 characters for your description, so try to expand on ideas introduced in the proposal's title, keeping in mind that a good chunk of ERC readers come from a technical background (and not necessarily financial.) For example, what is open/transparent financial data?

Reply:
You are right. I changed it to:
> Open Transparent Financial Data such as Revenue, Costs of Goods Sold, Operating Expenses, EBITDA and Earnings Per Share for all Investors.